be/src/vec/core/block.cpp

```diff
@@ -402,6 +402,20 @@ std::string Block::dump_data(size_t begin, size_t row_limit) const {
     return out.str();
 }
 
+std::string Block::dump_one_line(size_t row, int column_end) const {
+    assert(column_end < columns());
+    fmt::memory_buffer line;
+    for (int i = 0; i < column_end; ++i) {
+        if (LIKELY(i != 0)) {
+            // TODO: need more effective function of to string. now the impl is slow
+            fmt::format_to(line, " {}", data[i].to_string(row));
```

Comment:
why not check whether the data[i] is validity like this "  if (data[i].column) { s = data[i].to_string(row_num); } " ?